### PR TITLE
More info for Linux Mint regarding probe-run and SWD probe.

### PR DESCRIPTION
**Pull request by NollKollTroll**
Added some info to make it easier installing probe-run and SWD probe on a Linux Mint system.

### CODE REVIEW

File: ASSEMBLY.md [hunk 1]
```diff
@@ -85,9 +85,11 @@ The Board Management Controller (BMC) is responsible for controlling power and r
 3. Ensure you have the Rust Programming Language installed (see <https://www.rust-lang.org>)
 4. Install the `probe-run` tool:
 
-```console
-$ cargo install probe-run
-```
+	```console
```
Comment (thejpster) [inline ASSEMBLY.md:88]: Unsure what this indentation is for?

File: ASSEMBLY.md [hunk 2]
```diff
@@ -106,9 +108,9 @@ $ cargo install probe-run
 2. Ensure you have the Rust Programming Language installed (see <https://www.rust-lang.org>)
 3. Install the `probe-run` tool (if you didn't do it earlier):
 
-```console
-$ cargo install probe-run
-```
+	```console
```
Comment (thejpster) [inline ASSEMBLY.md:111]: Unsure what this indentation is for?

File: ASSEMBLY.md [hunk 3]
```diff
@@ -124,6 +126,13 @@ Time to learn Rust and get hacking on the BIOS and the OS! That's what the Neotr
 
 You can make one out of a Raspberry Pi Pico. See <https://github.com/rp-rs/rp2040-project-template/blob/main/debug_probes.md> for more details.
 
+For Linux Mint, the udev-rules also needs to be updated for the SWD debug probe. The file `/etc/udev/rules.d/99-cmsis-dap.rules` should include:
+
+```
+# cafe:4005 RP2040 Pico with CMSIS-DAP
+SUBSYSTEM=="usb", ATTR{idVendor}=="cafe", ATTR{idProduct}=="4005", MODE:="0666"
+```
```
Comment (thejpster) [inline ASSEMBLY.md:134]: Add a note here to adjust the rule according to the VID and PID of the programmer being used.

File: ASSEMBLY.md [hunk 4]
```diff
@@ -124,6 +126,13 @@ Time to learn Rust and get hacking on the BIOS and the OS! That's what the Neotr
 
 You can make one out of a Raspberry Pi Pico. See <https://github.com/rp-rs/rp2040-project-template/blob/main/debug_probes.md> for more details.
 
+For Linux Mint, the udev-rules also needs to be updated for the SWD debug probe. The file `/etc/udev/rules.d/99-cmsis-dap.rules` should include:
```
Comment (thejpster) [inline ASSEMBLY.md:129]:
```suggestion
For Linux Mint (and likely most other modern Linux distributions), the udev-rules also need to be updated for the SWD debug probe. The file `/etc/udev/rules.d/99-cmsis-dap.rules` should be created, containing:
```

**Review by thejpster — CHANGES_REQUESTED** [review]

File: ASSEMBLY.md (hunk 1 above)
Comment (NollKollTroll) [reply ASSEMBLY.md]: Elements in lists need to be indented according to https://www.markdownguide.org/basic-syntax/ under "Adding Elements in Lists"

File: ASSEMBLY.md (hunk 3 above)
Comment (NollKollTroll) [reply ASSEMBLY.md:135]: Done

File: ASSEMBLY.md (hunk 4 above)
Comment (NollKollTroll) [reply ASSEMBLY.md]: Agree

File: ASSEMBLY.md (hunk 2 above)
Comment (NollKollTroll) [reply ASSEMBLY.md]: Elements in lists need to be indented according to https://www.markdownguide.org/basic-syntax/ under "Adding Elements in Lists"

**Comment by NollKollTroll** [comment]
I read https://www.markdownguide.org/basic-syntax/ again and I have tried to use the best practices to update the document as a whole. Mainly regarding elements in lists to get them to render unambiguously.

**Comment by NollKollTroll** [comment]
No not in progress, just more edits because of your justified remarks that made me question how markdown works. I am done, until/unless you come with more remarks.

**Review by thejpster — APPROVED** [review]

**Comment by thejpster** [comment]
Ah, I see the need for the whitespace now. Thank you for the changes!

**Pull request merged**